static/app/components/forms/field/index.tsx

```diff
@@ -171,6 +171,11 @@ function Field({
     Control = <FieldControl {...controlProps}>{children}</FieldControl>;
   }
 
+  // Provide an `aria-label` to the FieldDescription label if our label is a
+  // string value. This helps with testing and accessability. Without this the
+  // aria label contains the entire description.
+  const ariaLabel = typeof label === 'string' ? label : undefined;
```

Comment:
this is super nice... should we add an optional `aria-label` prop too?

Reply:
Yeah added something like that now. `labelText`